packages/schema/src/schema/processing/dissolve.json

```diff
@@ -9,6 +9,10 @@
       "type": "string",
       "description": "The input layer for the dissolve operation."
     },
+    "outputLayerName": {
+      "type": "string",
+      "description": "The name of the output layer for the buffered geometry."
+    },
```

Comment:
We should not change the schemas for implementing this feature, we already have the name as part of the jgis file schema.

I'd suggest you look at how we allow setting a name in the layer forms implementation.